src/website/views/components/DropDown/DropdownMenu.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react';
+import styles from './dropdown.module.css';
+import { connect } from 'react-redux';
+import { DndContext, closestCenter } from '@dnd-kit/core';
+import {
+  SortableContext,
+  verticalListSortingStrategy,
+  useSortable,
+  arrayMove,
+} from '@dnd-kit/sortable';
+import {
+  restrictToVerticalAxis,
+  restrictToWindowEdges,
+} from '@dnd-kit/modifiers';
+import { CSS } from '@dnd-kit/utilities';
+
+const SortableGraph = ({ id, children, order, dispatch, setOrder }) => {
+
+  const { attributes, listeners, setNodeRef, transform, transition } = useSortable({
+    id,
+  })
+
+  const style = {
+    transition,
+    transform: CSS.Transform.toString(transform),
+  }
+
+  // const handleRightClick = (event) => {
+  //   event.preventDefault();
+
+  //   function removeGraph(graphInOrder) {
+  //     console.log(id + " was clicked")
+  //     return (id !== graphInOrder)
+  //   }
+
+  //   let filteredOrder = order.filter(removeGraph)
+  //   console.log(filteredOrder)
+  //   dispatch({type: 'REARRANGE_GRAPHS', payload: filteredOrder})
+  //   setOrder(filteredOrder)
+  // }
+
+  return (
+    <div
+      ref={setNodeRef}
+      style={style}
+      {...attributes}
+      {...listeners}
+      className={styles.dropdownItem}
+      // onContextMenu={handleRightClick}
+      suppressHydrationWarning={true} // I know this is irresponsible but the warning is very annoying and i have no idea how to fix it ill ask one of you later
```

Comment:
We can keep this option. Was this related to the error: `Hydration failed because the initial UI does not match what was rendered on the server.`?

Please make sure to also remove the comment.